### PR TITLE
sh

### PR DESCRIPTION
**Pull request by muhammadahmad665**

### CODE REVIEW

File: src/ncd_nft/build.sh [hunk 1]
```diff
@@ -1,3 +1,41 @@
 #!/bin/bash
 cd "$(dirname "$0")"
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
+cd "$(dirname "$0")"
```
Comment (Azizhamdani7) [inline src/ncd_nft/build.sh]: what have you done here, it was supposed like this.

Comment (muhammadahmad665) [reply src/ncd_nft/build.sh]: avi sir

File: src/ncd_nft/build.sh [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
+cd "$(dirname "$0")"
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
+cd "$(dirname "$0")"
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
+cd "$(dirname "$0")"
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
+cd "$(dirname "$0")"
+cargo build --all --target wasm32-unknown-unknown --release#!/bin/bash
```
Comment (muhammadahmad665) [inline src/ncd_nft/build.sh]: you can see here

**Review by Azizhamdani7 — CHANGES_REQUESTED** [review]
sdsa

File: src/ncd_nft/build.sh (hunk 1 above)
Comment (muhammadahmad665) [reply src/ncd_nft/build.sh]: working on it

**Review by Azizhamdani7 — APPROVED** [review]
Approved.

**Pull request closed without merging**